pandas/tests/dtypes/test_inference.py

```diff
@@ -39,6 +39,11 @@
 from pandas.util import testing as tm
 
 
+@pytest.fixture(params=[True, False], ids=lambda val: str(val))
+def coerce(request):
+    return request.param
```

Comment:
Should we do this as a parameterize instead of fixture? (I don't know whether are some 'rules' where it is better to use one or the other, but in this case it really seems to be a parameterizing of the test, so I would find it easier to read the code if it uses paramatrize)

Reply:
I use fixture whenever I find myself using the same parametrization multiple times.  As you can see below, I use it three times.